packages/kbn-unified-field-list/src/containers/unified_field_list_item/field_list_item.tsx

```diff
@@ -232,6 +239,9 @@ function UnifiedFieldListItemComponent({
 }: UnifiedFieldListItemProps) {
   const [infoIsOpen, setOpen] = useState(false);
 
+  const isBreakdownSupported =
```

Comment:
Another improvement we could do (not necessary for this PR), is to show the breakdown action if the current field is non-aggregatable but it has an aggregatable subfield which we could still use:

<img width="782" alt="Screenshot 2024-11-11 at 17 37 47" src="https://github.com/user-attachments/assets/db22ca2e-8ca5-42fa-90b6-448306570c4a">